### ISSUE TITLE
Processing stops on date field

### ISSUE THREAD

**Issue by antalk2** [linville/pydbisam#3]
type id: 2 (date), 4 bytes
An option to skip date fields  would be nice.
(I only saw date fields filled with zero bytes, so cannot check decoding)

*Observations on the file format*

1. Apparently before each field value there is a byte with 00 or 01 as value.

(The first of these in a row is mentioned in
https://github.com/linville/pydbisam/blob/main/NOTES.md
under *Row Definition*:   (Offset: 0x19 Size: 2  Description: Trailing \x01 marker),
however it is only 1 byte)

- When the byte is 00, the field value is filled with zeros.
- My guess: 00 encodes not set (NULL) values to distinguish from actual zeros. 

2. Under *File format*: the "MD5 Signature" is identical among a set of files I saw. So probably not a checksum.

**Comment by antalk2** [comment]
I see there is: https://github.com/linville/pydbisam/compare/empty-field-marker , 
so (1.) above is already known.

**Comment by linville** [comment]
Thanks for the report!

I added support for date fields. It's unverified as I don't have any databases with that type to verify against but I think it's likely to work. I'll release a new version here shortly, let me know here if you experience any issues.

I merged in the `empty-field-marker` branch (just forgot it was out there waiting).

Regarding the 16-byte field in the header, yes I'm seeing that of the few databases I have, they all have the same value too. I've changed the notes to be more ambiguous since I don't have any good docs on what it is. A windows utility that displays the databases refers to that field as "File Signature" so I've just updated the file notes to reflect as-such.
